src/css/specific/notice-banner.css

```diff
@@ -0,0 +1,35 @@
+.notice-banner {
```

Comment:
Can you make sure it's left-aligned?

Reply:
it is left-aligned, please see attached video to validate.

I have removed all the images of the earlier designs (which are center-aligned) to avoid confusion. The video is now the source of truth